### ISSUE TITLE
Slider: Focus on slider on mouse click not working

### ISSUE THREAD

**Issue by kilobyte2007** [primefaces/primevue#5446]
### Describe the bug

The issue mentioned in #3748 is present again.
It's reproduceable on the docs site - https://primevue.org/slider/

### Reproducer

https://primevue.org/slider/

### PrimeVue version

3.50.0

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Vue CLI App

### Browser(s)

Chrome 122.0.6261.129

### Steps to reproduce the behavior

Go to https://primevue.org/slider/ and focus the slider with the mouse - it's not being focused at all. Not being able to move with the arrow keys as well.

### Expected behavior

The slider handle should be visibly focused and accessible even on mouse click.
